src/components/MessageBox.js

```diff
@@ -22,13 +24,18 @@ class MessageBox extends React.Component {
     title: PropTypes.string
   };
 
+  static defaultProps = {
+    ariaLive: 'region',
+    contentRole: 'status',
+  }
+
   render () {
     const theme = StyleUtils.mergeTheme(this.props.theme);
     const styles = this.styles(theme);
 
     return (
       <article
-        aria-live='polite'
+        aria-live={this.props.ariaLive}
         className='mx-message-box'
```

Comment:
It looks like the default was 'polite' and it is now being set to 'region' for the aria-live attribute. I'm worried this may break something that was depending on 'polite' for this attribute?

Reply:
Missed this completely. Nice catch.